src/static.js

```diff
@@ -37,7 +37,7 @@ export const exportRoutes = async ({ config, clientStats, cliArguments }) => {
       }
 
       // Loop through the props
-      Object.values(route.initialProps).forEach(prop => {
+      Object.keys(route.initialProps).map(k => route.initialProps[k]).forEach(prop => {
```

Comment:
Ignore this line in this diff. It is part of the PR #258 that this PR depends on. Once that is merged, this will disappear from the diff.